pajbot/apiwrappers/twitch/helix.py

```diff
@@ -349,7 +349,9 @@ def _fetch_subscribers_page(
         # }
 
         subscribers = [
-            UserBasics(entry["user_id"], entry["user_login"], entry["user_name"]) for entry in response["data"]
+            UserBasics(entry["user_id"], entry["user_login"], entry["user_name"])
+            for entry in response["data"]
+            if entry["user_login"]
```

Comment:
Would appreciate a more verbose check here, e.g. `if entry["user_login"] != ""`, and please place a comment indicating *why* this check is here, with a short summary and a link to this PR and/or the TwitchDev issue

Reply:
Good thinking; updated in https://github.com/pajbot/pajbot/pull/2292/commits/c51dcdb4bd188c075250b046481b007cced6e1fc